src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -745,35 +745,41 @@ export class VolumesListTableConfig implements InputTableConf {
             name: T('Upgrade Pool'),
             label: T("Upgrade Pool"),
             onClick: (row1) => {
-              this.dialogService
-                .confirm(
-                  T("Upgrade Pool"),
-                  helptext.upgradePoolDialog_warning + row1.name
-                )
-                .subscribe(confirmResult => {
-                  if (confirmResult === true) {
-                    this.loader.open();
-                    this.ws.call("pool.upgrade", [rowData.id]).subscribe(
-                      res => {
-                        this.dialogService
-                          .Info(
-                            T("Upgraded"),
-                            T("Successfully Upgraded ") + row1.name
-                          )
-                          .subscribe(infoResult => {
-                            this.parentVolumesListComponent.repaintMe();
+              this.translate.get(helptext.upgradePoolDialog_warning).subscribe(warning => {
+                this.dialogService
+                  .confirm(
+                    T("Upgrade Pool"),
+                      warning + row1.name
+                  )
+                  .subscribe(confirmResult => {
+                    if (confirmResult === true) {
+                      this.loader.open();
+                      this.ws.call("pool.upgrade", [rowData.id]).subscribe(
+                        res => {
+                          this.translate.get(T("Successfully Upgraded ")).subscribe(success_upgrade => {
```

Comment:
Is the extra space in (T("Successfully Upgraded ")) necessary?

Reply:
yea because the pool name comes after